project/First-Project/src/main/java/first/project/service/AnswerServiceImpl.java

```diff
@@ -0,0 +1,73 @@
+package first.project.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+
+import first.project.dto.AnswerDTO;
+import first.project.exceptions.AddAnswerException;
+import first.project.model.Answer;
+import first.project.model.Question;
+import first.project.repository.AnswerRepository;
+import first.project.repository.QuestionRepository;
+
+@Service
+public class AnswerServiceImpl implements AnswerService
+{
+    @Autowired
+    private AnswerRepository answerRepository;
+
+    @Autowired
+    private QuestionRepository questionRepository;
+
+    @Override
+    public boolean checkEmail(String email)
+    {
+        if ("".equals(email))
+        {
+            return false;
+        }
+        return true;
+    }
+
+    @Override
+    public boolean checkAnswer(String answer)
+    {
+        if ("".equals(answer))
+        {
+            return false;
+        }
+        return true;
+    }
+
+    @Override
+    public Answer validateAndPost(AnswerDTO answerDTO) throws AddAnswerException
```

Comment:
Rename the method to `validateAndSave`, validateAndPost name is misleading and will trick you into thinking that you're trying to make some Post Requst